src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -1274,37 +1132,45 @@ export const selectTotalBondingsBalanceByAssetId = createSelector(
 // We need to explicitly deep output compare, since ([SHAPESHIFT_VALIDATOR_ADDRESS] === [SHAPESHIFT_VALIDATOR_ADDRESS]) === false
 export const selectValidatorIdsByFilter = createDeepEqualOutputSelector(
   selectPortfolioAccounts,
-  selectAccountSpecifierParamFromFilterOptional,
-  (portfolioAccounts, accountSpecifier): PubKey[] => {
-    if (!accountSpecifier)
+  selectAccountIdParamFromFilterOptional,
+  (portfolioAccounts, accountId): PubKey[] => {
+    if (!accountId)
       return uniq(
         values(portfolioAccounts).flatMap(portfolioAccount => portfolioAccount.validatorIds ?? []),
       )
-    const portfolioAccount = portfolioAccounts?.[accountSpecifier]
+    const portfolioAccount = portfolioAccounts?.[accountId]
     if (!portfolioAccount) return []
     if (!portfolioAccount?.validatorIds?.length)
-      return [getDefaultValidatorAddressFromAccountId(accountSpecifier)]
+      return [getDefaultValidatorAddressFromAccountId(accountId)]
 
     return portfolioAccount.validatorIds
   },
 )
 
 const selectDefaultStakingDataByValidatorId = createSelector(
-  selectAssetIdParamFromFilterOptional,
-  selectSupportsCosmosSdkParamFromFilterOptional,
+  selectAssetIdParamFromFilter,
   selectValidators,
-  (assetId, supportsCosmosSdk = true, stakingDataByValidator) => {
```

Comment:
this supportsCosmosSdk bizness was causing a headache, and the changes yesterday effectively made this redundant